clevertap-core/src/main/java/com/clevertap/android/sdk/ControllerManager.java

```diff
@@ -195,11 +197,16 @@ private void _initializeInbox() {
                 return;
             }
             if (deviceInfo.getDeviceID() != null) {
-                setCTInboxController(new CTInboxController(config, deviceInfo.getDeviceID(),
-                        baseDatabaseManager.loadDBAdapter(context),
-                        ctLockManager,
-                        callbackManager,
-                        Utils.haveVideoPlayerSupport));
+                setCTInboxController(
+                        new CTInboxController(
+                                config,
+                                deviceInfo.getDeviceID(),
+                                baseDatabaseManager.loadDBAdapter(context),
+                                ctLockManager,
+                                callbackManager,
+                                VideoLibChecker.INSTANCE.haveVideoPlayerSupport
+                        )
```

Comment:
Why not VideoLibChecker.haveVideoPlayerSupport

Reply:
done